lib/perl/Genome/Model/Tools/Bsmap/MethylationRatio.pm

```diff
@@ -0,0 +1,99 @@
+package Genome::Model::Tools::Bsmap::MethylationRatio;
+
+use strict;
+use warnings;
+use Cwd;
+use Genome;
+
+use Genome::Utility::Text qw(sanitize_string_for_filesystem);
+use File::Spec;
+
+my $METHRATIO_COMMAND = File::Spec->join(getcwd, 'methylation_ratio.py');
```

Comment:
This path needs to be relative to the module's directory rather than the current working directory.  Something like 
```perl
use File::Basename qw();
my (undef, $dir) = File::Basename::fileparse(__FILE__);
my $METHRATIO_COMMAND = File::Spec->join($dir, 'methylation_ratio.py');
```